spec/controllers/url_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe UrlsController do
```

Comment:
As far as I remember there's no need to use `RSpec.describe` as opposed to simply `describe` here